lang-ca.js

```diff
@@ -924,9 +930,11 @@ SnapTranslator.dict.ca = {
     'add comment':
         'afegeix un comentari',
     'undrop':
-        'recupera bloc',
+        'desfés',
     'undo the last\nblock drop\nin this pane':
-        'recupera l\'últim bloc\nque s\'hagi llençat',
+        'desfés l\'últim moviment\nde blocs',
```

Comment:
same here, the original translation conveys better what the action does. "Undrop" does not "undo" the latest block movement, that's what "undo" does. In any case, maybe it should have been phrased "recupera l\'últim bloc\nque s\'hagi deixat anar".